### PR TITLE
Sodn yang orderless

### PR DESCRIPTION
**Pull request by sowmyadn010501**
updated orderless config like below mobile-network at depth 5.
```
mobile-network v6pool 1000::1 pool-prefix 64 network-prefix 64
mobile-network pool 2.2.2.255 pool-prefix 8 network-prefix 8
```
updated orderless config like below ospfv3 neighbor  at depth 1.
```
- ospfv3 neighbor FE80::AC5 priority 100 poll-interval 100 cost 100
- ospfv3 neighbor FE80::AC4 priority 1 poll-interval 1 database-filter all out
- ospfv3 1 ipv4 neighbor FE80::AC5 priority 100 poll-interval 100 cost 100
```
Pass log1: https://earms-trade.cisco.com/tradeui/logs/details?archive=/nobackup/vijviswa/cmt_script/polaris/binos/linkfarm/sdks-host/iosxe-pyats/nativesdk/sdk/sysroots/x86_64-xesdk-linux/usr/users/vijviswa/archive/24-06/cli2show_job.2024Jun18_12:37:42.230880.zip&atstype=ATS

Pass log2: https://earms-trade.cisco.com/tradeui/logs/details?archive=/nobackup/krajagan/polaris_keerthy/polaris/binos/linkfarm/sdks-host/iosxe-pyats/nativesdk/sdk/sysroots/x86_64-xesdk-linux/usr/users/krajagan/archive/24-06/cli2show_job.2024Jun25_07:44:32.890224.zip&atstype=ATS

### CODE REVIEW

**Review by yuekyang — COMMENTED** [review]
Could you attach a passing log? Thanks.

**Comment by sowmyadn010501** [comment]
> Could you attach a passing log? Thanks.

Attached.

File: ncdiff/src/yang/ncdiff/runningconfig.py [hunk 1]
```diff
@@ -97,6 +97,11 @@
     (re.compile(r'^ *mpls mldp static '), 0),
     (re.compile(r'^ *router '), 0),
     (re.compile(r'^ *summary-address '), 1),
+    (re.compile(r'^ *mobile-network '), 4),
+    (re.compile(r'^ *ospfv3 neighbor '), 1),
+    (re.compile(r'^ *ospfv3 1 ipv(\d) neighbor'), 1),
```
Comment (yuekyang) [inline ncdiff/src/yang/ncdiff/runningconfig.py:102]: What is this `1` in `ospfv3 1 ipv4 neighbor`? Is it a process-id? I need a snip of running-config to understand its meaning.

File: ncdiff/src/yang/ncdiff/runningconfig.py [hunk 2]
```diff
@@ -97,6 +97,11 @@
...
+    (re.compile(r'^ *area '), 1),
+    (re.compile(r'^ *ospfv3 1 neighbor '), 1),
```
Comment (yuekyang) [inline ncdiff/src/yang/ncdiff/runningconfig.py:104]: What is this `1` in `ospfv3 1 neighbor`?

File: ncdiff/src/yang/ncdiff/runningconfig.py (hunk 1 above)
Comment (sowmyadn010501) [reply ncdiff/src/yang/ncdiff/runningconfig.py:102]: Hi,
This is the output for this
```
290099:  -   ospfv3 1 ipv6 neighbor FE80::AC5 priority 100 poll-interval 100 cost 100
290100:  -   ospfv3 1 ipv6 neighbor FE80::AC4 priority 1 poll-interval 1 database-filter all out
290101:  -   ospfv3 1 ipv6 neighbor FE80::AC3 priority 11 cost 1 database-filter all out
290102:  -   ospfv3 1 ipv6 neighbor FE80::AC2 poll-interval 1 cost 1 database-filter all out
290103:  -   ospfv3 1 ipv6 neighbor FE80::AC1 priority 25 cost 10
290104:  -   ospfv3 1 ipv6 neighbor FE80::AB9 priority 25 poll-interval 65535
290105:  -   ospfv3 1 ipv6 neighbor FE80::AB1 priority 10 database-filter all out
290106:  -   ospfv3 1 ipv6 neighbor FE80::AB2 poll-interval 255 database-filter all out
290107:  -   ospfv3 1 ipv6 neighbor FE80::AB3 cost 10 database-filter all out
290108:  -   ospfv3 1 ipv6 neighbor FE80::AB4 priority 255
290109:  -   ospfv3 1 ipv6 neighbor FE80::AB5 poll-interval 65535
290110:  -   ospfv3 1 ipv6 neighbor FE80::AB6 database-filter all out
290111:  -   ospfv3 1 ipv6 neighbor FE80::AB7 cost 1
```
and here is the pass log : https://earms-trade.cisco.com/tradeui/logs/logfile?logfile=TaskLog.Task-1&begin=27157921&size=853292&archive=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr%2Fusers%2Fvijviswa%2Farchive%2F24-06%2Fcli2show_job.2024Jun19_08:53:05.637932.zip&ats=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr&submitter=vijviswa&from=trade&view=all&atstype=pyATS

File: ncdiff/src/yang/ncdiff/runningconfig.py (hunk 2 above)
Comment (sowmyadn010501) [reply ncdiff/src/yang/ncdiff/runningconfig.py:104]: Hi, i have updated the pass log in above comment and this below log is exact where the issue seen
https://earms-trade.cisco.com/tradeui/logs/logfile?logfile=TaskLog.Task-1&begin=21919020&size=820013&archive=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr%2Fusers%2Fvijviswa%2Farchive%2F24-06%2Fcli2show_job.2024Jun19_08:53:05.637932.zip&ats=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr&submitter=vijviswa&from=trade&view=all&atstype=pyATS

**Pull request closed without merging**